interfaces/Referral-App/src/app/help/help.page.html

```diff
@@ -17,25 +17,39 @@
         <p>
           <ion-icon class="help-call-icon" name="call"></ion-icon> {{
           help.helpPhoneLabel }}:
-          <a href="tel:{{ help.helpPhone }}">{{ help.helpPhone }}</a>
+          <a href="tel:{{ help.helpPhone }}" (click)="logClick('phone')"
+            >{{ help.helpPhone }}</a
+          >
         </p>
       </ion-col>
     </ion-row>
     <ion-row>
       <ion-col>
-        <a href="{{ help.helpWhatsApp }}" *ngIf="help.helpWhatsApp">
+        <a
+          href="{{ help.helpWhatsApp }}"
```

Comment:
For WhatsApp its possible to create a smoothe 'app link' by using the URL https://wa.me/31600000000
If PA have the app installed, it will automatically open, others will land on a (sort of) helpful page.

(And for these external links I would also suggest to add the `target` and `rel`-attributes.)

Reply:
the href value can be set via the google sheet, new tab attributes are added